doc_src/fish_indent.txt

```diff
@@ -7,12 +7,16 @@ fish_indent [OPTIONS]
 
 \subsection fish_indent-description Description
 
-`fish_indent` is used to indent a piece of fish code. `fish_indent` reads commands from standard input and outputs them to standard output.
+`fish_indent` is used to indent a piece of fish code. `fish_indent` reads commands from standard input and outputs them to standard output or a specified file.
 
 The following options are available:
 
 - `-d` or `--dump` dumps information about the parsed fish commands to stderr
 
+- `-w` or `--write` indents a specified file and immediatly writes to that file
```

Comment:
@terlar, @krader1961 is talking about -W, not -w. I don't really care either way about -W, should I keep it or remove it?